Trimble.Modus.Components/Controls/ListView/TMListView.xaml.cs

```diff
@@ -0,0 +1,130 @@
+using System.Collections;
+using Trimble.Modus.Components.Enums;
+
+namespace Trimble.Modus.Components;
+
+public partial class TMListView : ContentView
+{
+    #region Bindable Properties  
+    public static readonly BindableProperty ItemsSourceProperty =
+        BindableProperty.Create(nameof(ItemsSource), typeof(IEnumerable), typeof(TMListView), null, propertyChanged: OnItemSourceChanged);
+
+    public static readonly BindableProperty ItemTemplateProperty =
+        BindableProperty.Create(nameof(ItemTemplate), typeof(DataTemplate), typeof(TMListView));
+
+    public static readonly BindableProperty SelectionModeProperty =
+             BindableProperty.Create(nameof(SelectionMode), typeof(ListSelectionMode), typeof(TMListView));
+    #endregion
+    #region Public properties  
+    public event EventHandler<SelectableItemEventArgs> ItemSelected;
```

Comment:
Can you rename this to 'SelectionChanged'? as the user can either single and multiple items.

Reply:
Refer CollectionView for example

Reply:
using object now